src/generic_layers/reduced_points.rs

```diff
@@ -73,7 +73,7 @@ impl<P: Reducible, const SIZE: u8, const SALT: u64> Chunk for ReducedUniformPoin
                             .then_with(|| p.position().cmp(&other.position()))
                             .is_lt();
 
-                    if other.position().manhattan_dist(p.position()) < p.radius() + other.radius()
+                    if other.position().manhattan_dist(p.position()) < p.radius()

```

Comment:
My intent was to avoid points that overlap, not points that include the others' center. I'll need to think about what you wrote here